.vib/tomcat/cypress/cypress/integration/tomcat_spec.js

```diff
@@ -0,0 +1,16 @@
+/// <reference types="cypress" />
+import { random } from '../support/utils';
+
+it('allows to deploy a new WAR app', () => {
+  cy.visitAuth(`/manager`);
+  cy.contains('Web Application Manager');
+  cy.contains('form', 'Select WAR file').within(() => {
```

Comment:
Looking into simplifying the logic here, is this `contains` necessary? Could we make it work leaving only the `cy.get('[type="file"]')` below? Usually, there are no multiple elements with that selector.

Reply:
It is true that there is only a single `[type="file"]`, but there are more than one `Deploy` buttons as you can see: 

![deploy_form](https://user-images.githubusercontent.com/24253844/185913972-84557462-a61a-45a4-bc22-7267644bae3d.png)

The `form` selector is meant for that specific `Deploy` button, and not the `[type="file"]` field 😁